### DISCUSSION TITLE
MITMing the policy on a Chromebook

### DISCUSSION THREAD

**Discussion by ingoau** [3kh0/ext-remover#1179] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I had this idea to MITM the proxy on a Chromebook using dns rewrites. If I rewrite dm.google.com to my proxy server (10.1.3.1), I can change the policy. The problem is SSL, but I should be able to trust my CA.

Updates: https://github.com/3kh0/ext-remover/discussions/1179#discussioncomment-10781296

**Comment by ingoau** [comment]
I will update here with any progress

Note: this is not an exploit or guide, just here for information purposes

**Reply by ingoau** [reply]
Just discovered its m.google.com

**Reply by ingoau** [reply]
Damn. It shows as trusted when I access it but policy not updating and no logs. It's too secure....

**Reply by ingoau** [reply]
Just fyi here's how I did this:
1. Create a CA on Opnsense firewall (you can prob do it another way but easiest)
2. Create a combined client/server cert
3. Trust the CA on target
4. Set up some kind of DNS (Im using adguard)
5. Rewrite m.google.com to proxy server or computer (I am using a nginx proxy server at 10.1.3.1)
6. Add the certs
![image](https://github.com/user-attachments/assets/3eb08bba-5834-41fa-a074-3a64047b8c09)
7. Create proxy host to a machine running a server in your network
![image](https://github.com/user-attachments/assets/7f3ef281-a5c7-41aa-a3b8-34b4cbb0edf3)
![image](https://github.com/user-attachments/assets/224d9da2-096b-4bb6-8423-73da3fdbb2d2)
8. Start a server on the machine your proxy server/dns is pointed at (I just used `python3 http.server`), preferably with logs
9. Navigate to m.google.com to test
10. Reload policy
11. Realize custom certs are not trusted :(

**Reply by ghost** [reply]
I'm confused can you help me

**Reply by ingoau** [reply]
> I'm confused can you help me
> 

Don't do this if you don't know what you are doing. I was just putting this here for more advanced people just so they know it isn't possible. The way I did it involved my own infrastructure (server) and Linux and networking knowledge. Basically this isn't a solution to anything or a way to change policy, it is just records of what have been tried.
